src/renderer/account/components/SendPanel/SendPanel.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
-import { func, string, bool } from 'prop-types';
+import { func, string, bool, objectOf } from 'prop-types';
 import { wallet } from '@cityofzion/neon-js';
 import { BigNumber } from 'bignumber.js';
-import { map, noop } from 'lodash';
+import { noop, values, mapValues, keyBy } from 'lodash';
 
 import Panel from 'shared/components/Panel';
 import Button from 'shared/components/Forms/Button';
 import Input from 'shared/components/Forms/Input';
 import Icon from 'shared/components/Icon';
 import Select from 'shared/components/Forms/Select';
-import { ASSETS, NEO } from 'shared/values/assets';
+import { NEO } from 'shared/values/assets';
 
 import styles from './SendPanel.scss';
 import isNumeric from '../../util/isNumeric';
 
+import balanceShape from '../../shapes/balanceShape';
```

Comment:
:neckbeard: No need for the extra whitespace above this line, it can be grouped with the other relative imports.